pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -526,8 +526,8 @@ def _process_weekly_object_to_country_time_series(repw_res, meta_glob, regions_h
         for regid, regname in region_ids.items():
             ts_data = {
                 "time": time,
-                "seasonal": {"obs": yeardict, "mod": yeardict},
-                "yearly": {"obs": yeardict, "mod": yeardict},
+                "seasonal": {"obs": deepcopy(yeardict), "mod": deepcopy(yeardict)},
+                "yearly": {"obs": deepcopy(yeardict), "mod": deepcopy(yeardict)},
```

Comment:
see my previous comment about `collections.defaultdict`.
Also, how did it work before the `deepcopy`?
Is this cover by any tests?